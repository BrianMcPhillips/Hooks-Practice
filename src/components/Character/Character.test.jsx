import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Character from './Character';

describe('Character component', () => {
  afterEach(() => cleanup());
  it('renders Character', () => {
    const { asFragment } = render(
      <Character
        name="arnold"
        image="https://vignette.wikia.nocookie.net/heyarnold/images/f/f5/Arnold.png/revision/latest?cb=20181027162333"
      />);
    expect(asFragment()).toMatchSnapshot();
  });
});
