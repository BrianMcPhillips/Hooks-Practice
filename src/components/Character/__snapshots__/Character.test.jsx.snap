// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character component renders Character 1`] = `
<DocumentFragment>
  <img
    alt="arnold"
    src="https://vignette.wikia.nocookie.net/heyarnold/images/f/f5/Arnold.png/revision/latest?cb=20181027162333"
  />
  <h3>
    arnold
  </h3>
</DocumentFragment>
`;
